test/fields/collections/Upload/e2e.spec.ts

```diff
@@ -81,11 +81,41 @@ describe('Upload', () => {
     await saveDocAndAssert(page)
   }
 
-  // eslint-disable-next-line playwright/expect-expect
   test('should upload files', async () => {
     await uploadImage()
   })
 
+  test('should upload files from remote URL', async () => {
+    await uploadImage()
+
+    await page.goto(url.create)
+
+    const pasteURLButton = page.locator('.file-field__upload .dropzone__file-button', {
+      hasText: 'Paste URL',
+    })
+    await pasteURLButton.click()
+
+    const remoteImage =
+      'https://fastly.picsum.photos/id/11/2500/1667.jpg?hmac=xxjFJtAPgshYkysU_aqx2sZir-kIOjNR9vx0te7GycQ'
```

Comment:
Do we think this URL will be available long-term? If not, we might want to use something in our control like the og-image from our site.

Reply:
I was thinking the same thing - I will update